keps/sig-node/2400-node-swap/README.md

```diff
@@ -117,6 +118,19 @@ support to nodes in a controlled, predictable manner so that Kubernetes users
 can perform testing and provide data to continue building cluster capabilities
 on top of swap.
 
+Since swap enablement is crucial to many different parties, this KEP aims to
+introduce basic swap enablement and leave further extensions to follow-up KEPs,
+that are already being worked on and will be introduced in the very near future.
+This way Kubernetes users / vendors would be able to use swap in a basic manner
+quickly while extensions would be brought to discussion in dedicated KEPs that
+would progress in the meantime.
+
+For example, to achieve this goal, this KEP does not introduce any new APIs.
```

Comment:
I think technically kubelet config fields are APIs, maybe `does not change the pod API`?

Reply:
Good point.
I wanted to emphasize that currently customizing how swap behaves is not really supported. You're right that setting `LimitedSwap` or `NoSwap` is an API, but it doesn't let you customize anything, but instead currently serves as an "on/off" API.

Changed to the following. WDYT?

```suggestion
For example, to achieve this goal, this KEP does not introduce any APIs
that allow customizing how the feature behaves, but instead only determines 
whether the feature is enabled or disabled.
```